src/main/java/com/jcabi/github/PullComment.java

```diff
@@ -77,6 +77,13 @@
     int number();
 
     /**
+     * Get its author.
+     * @return Pull comment author
+     * @throws IOException If there is any I/O problem
+     */
+    User author() throws IOException;
```

Comment:
Yes, that class :)
I don't think it defeats the purpose, you just use a decorator `PullComment.Smart(PullComment)` when you want to have the author. `Smart` classes are with extra features, that otherwise you would need json for.